pvacseq/lib/call_iedb.py

```diff
@@ -16,16 +16,19 @@ def main(args_input = sys.argv[1:]):
     parser.add_argument('output_file',
                         help="Output file from iedb")
     parser.add_argument('method',
-                        choices=pvacseq_utils.iedb_prediction_methods(),
+                        choices=PredictionClass.iedb_prediction_methods(),
                         help="The iedb analysis method to use")
     parser.add_argument('allele',
                         help="Allele for which to make prediction")
     parser.add_argument('epitope_length', type=int, choices=[8,9,10,11,12,13,14,15],
                         help="Length of subpeptides (epitopes) to predict")
     args = parser.parse_args(args_input)
-    pvacseq_utils.check_alleles_valid([args.allele])
-    pvacseq_utils.check_allele_valid_for_method(args.allele, args.method)
-    pvacseq_utils.check_length_valid_for_allele_and_method(args.epitope_length, args.allele, args.method)
+
+    PredictionClass.check_alleles_valid([args.allele])
+    prediction_class = globals()[PredictionClass.prediction_class_for_iedb_prediction_method(args.method)]
```

Comment:
I know next to nothing about python so maybe this is a dumb question, but can you just return a reference to the class directly rather than returning the string name of the class and then converting it back?